test/Dapr.AspNetCore.IntegrationTest/SubscribeEndpointTest.cs

```diff
@@ -70,7 +78,7 @@ public async Task SubscribeEndpoint_ReportsTopics()
 
                         if (element.TryGetProperty("route", out JsonElement route))
                         {
-                            subscriptions.Add((pubsubName, topic, route.GetString(), rawPayload, string.Empty, originalMetadataString));
+                            subscriptions.Add((pubsubName, topic, route.GetString(), rawPayload, string.Empty, originalMetadataString, deadLetterTopic));
```

Comment:
Is this manually adding the DLQ to the subscription?

Reply:
Discussed offline